app/src/main/java/org/simple/clinic/reassignPatient/ReassignPatientSheet.kt

```diff
@@ -130,6 +136,12 @@ class ReassignPatientSheet : BaseBottomSheet<
     override fun instantiateFragment() = ReassignPatientSheet()
   }
 
+  @Parcelize
+  data class SheetOpenedAndClosedFrom(
```

Comment:
Please rename it to `SheetClosed`. We are going to be using it in `PatientSummaryScreen` to get the result. So, we can have something like `ReassignPatientSheet.SheetClosed` as the screen result.